exercises/linked-list/example.py

```diff
@@ -45,3 +45,21 @@ def unshift(self, value):
             new_node.next = self.head
             self.head.prev = new_node
             self.head = new_node
+
+    def __len__(self):
```

Comment:
I think this implementation of `__len__` is cleaner than mine #408 but it requires O(n) to run when a call to `__len__` on a linked list could be O(1). Fortunately, this exercise doesn't mention anything about minimizing algorithmic complexity.